src/main/java/uk/gov/mca/beacons/api/configuration/SecurityConfiguration.java

```diff
@@ -19,6 +19,6 @@ protected void configure(HttpSecurity http) throws Exception {
 
   @Override
   public void configure(WebSecurity web) {
-    web.ignoring().antMatchers("/owner", "/actuator/health", "/actuator/info");
```

Comment:
Not sure why the `/owner` endpoint was ignored from our security config?

Reply:
looks like it was a temp thing to get the schema? @matthew-a-carr https://github.com/mcagov/beacons-service/commit/088869bcfac21fced9e6d43ea3d30f5df251b4a5

Reply:
Oh sorry that should be removed!